pkg/preparer/orchestrate_test.go

```diff
@@ -59,13 +60,34 @@ func testManifest(t *testing.T) *pods.PodManifest {
 	return manifest
 }
 
+type FakeStore struct{}
+
+func (f *FakeStore) Pod(string, string) (*pods.PodManifest, error) {
+	return nil, pods.NoCurrentManifest
+}
+
+func (f *FakeStore) SetPod(string, pods.PodManifest) (time.Duration, error) {
+	return 0, nil
+}
+
+func (f *FakeStore) RegisterPodService(pods.PodManifest) error {
+	return nil
+}
+
+func (f *FakeStore) WatchPods(string, <-chan struct{}, chan<- error, chan<- pods.PodManifest) error {
+	return nil
+}
+
 func TestPreparerLaunchesNewPodsThatArentInstalledYet(t *testing.T) {
 	testPod := &TestPod{
 		launchSuccess: true,
 	}
 	newManifest := testManifest(t)
 
-	var p *Preparer
+	p, _ := New("hostname", "0.0.0.0", "/hooks/dir", logging.DefaultLogger)
```

Comment:
This and the three subsequent lines keep getting repeated. Worth putting a small function in the test file that returns a test preparer?